CMakeLists.txt

```diff
@@ -5,14 +5,15 @@ list(APPEND SOURCES src/esp_littlefs.c src/littlefs_esp_part.c src/lfs_config.c)
 
 if(CONFIG_LITTLEFS_SDMMC_SUPPORT)
     list(APPEND SOURCES src/littlefs_sdmmc.c)
+    list(APPEND priv_requires sdmmc)
```

Comment:
I'm pretty sure `sdmmc` has to be in `pub_requires` because it is referenced in the public `include/esp_littlefs.h`

Reply:
Updated, PTAL~